Packs/CarbonBlackEnterpriseEDR/Integrations/CarbonBlackEnterpriseEDR/CarbonBlackEnterpriseEDR.py

```diff
@@ -0,0 +1,1325 @@
+        return f'ignore status for report with report_id "{report_id}" is false.'
+    else:
+        return f'ignore status for report with report_id "{report_id}" is true.'
+
+
+def remove_report_command(client: Client, args: dict) -> str:
+    report_id = args.get('report_id')
+    client.remove_report_request(report_id)
+
+    return f'The report "{report_id}" was deleted successfully.'
+
+
+def update_report_command(client: Client, args: dict) -> CommandResults:
+    report_id = args.get('report_id')
+    title = args.get('title')
+    description = args.get('description')
+    timestamp = convert_date_to_unix(args.get('timestamp'))
+    tags = argToList(args.get('tags'))
+    ipv4 = argToList(args.get('ipv4'))
+    ipv6 = argToList(args.get('ipv6'))
+    dns = argToList(args.get('dns'))
+    md5 = argToList(args.get('md5'))
+    ioc_query = argToList(args.get('ioc_query'))
+    severity = int(args.get('severity'))
+    ioc_contents = []
+
+    iocs = assign_params(
+        ipv4=ipv4,
+        ipv6=ipv6,
+        dns=dns,
+        md5=md5,
+        query=ioc_query
+    )
+
+    headers = ['ID', 'Title', 'Timestamp', 'Description', 'Severity', 'Link', 'IOCs_v2', 'Tags', 'Visibility']
+    result = client.update_report_request(report_id, title, description, severity, iocs, tags, timestamp)
+
+    contents = {
+        'ID': result.get('id'),
+        'Timestamp': convert_unix_to_timestamp(result.get('timestamp')),
+        'Description': result.get('description'),
+        'Title': result.get('title'),
+        'Severity': result.get('severity'),
+        'Tags': result.get('tags'),
+        'Link': result.get('link'),
+        'Visibility': result.get('visibility')
+    }
+
+    context = {
+        'ID': result.get('id'),
+        'Timestamp': convert_unix_to_timestamp(result.get('timestamp')),
+        'Description': result.get('description'),
+        'Title': result.get('title'),
+        'Severity': result.get('severity'),
+        'Tags': result.get('tags'),
+        'Link': result.get('link'),
+        'IOCs': result.get('iocs_v2'),
+        'Visibility': result.get('visibility')
+    }
+
+    iocs = result.get('iocs_v2', {})
+    for ioc in iocs:
+        ioc_contents.append({
+            'ID': ioc.get('id'),
+            'Match_type': ioc.get('match_type'),
+            'Values': ioc.get('values'),
+            'Field': ioc.get('field'),
+            'Link': ioc.get('link')
+        })
+
+    readable_output = tableToMarkdown(f'The report was updated successfully.', contents, headers, removeNull=True)
+    ioc_output = tableToMarkdown(f'The IOCs for the report', ioc_contents, removeNull=True)
+    results = CommandResults(
+        outputs_prefix='CarbonBlackEEDR.Report',
+        outputs_key_field='ID',
+        outputs=context,
+        readable_output=readable_output + ioc_output,
+        raw_response=result
+    )
+    return results
+
+
+def get_file_device_summary(client: Client, args: dict) -> CommandResults:
```

Comment:
why are we not outputting to File context standard here?

Reply:
Nothing in the information that return from this command, relevant to File context standard.
The only command that's relevant is `file_metadata`